rplugin/python3/LanguageClient/RPC.py

```diff
@@ -60,13 +60,25 @@ def notify(self, method: str, params: Dict[str, Any]) -> None:
     def serve(self):
         content_length = 0
         while not self.infile.closed:
-            line = self.infile.readline().decode("UTF-8").strip()
+            try:
+                data = self.infile.readline()
+                line = data.decode("UTF-8").strip()
+            except Exception:
+                msg = "Failed to decode message as UTF-8: " + data
+                logger.exception(msg)
+                continue
             if line:
                 header, value = line.split(":")
                 if header == "Content-Length":
                     content_length = int(value)
             else:
-                content = self.infile.read(content_length).decode("UTF-8")
+                try:
+                    data = self.infile.read(content_length)
+                    content = data.decode("UTF-8")
+                except Exception:
+                    msg = "Failed to decode message as UTF-8: " + data
```

Comment:
`+` cannot be used to concatenate string and bytes.

Reply:
So what should be done here?

Reply:
Could use `str(data)`

Reply:
I changed it to data.decode('utf-8', 'backslashreplace')

Reply:
I don't think change to `data.decode('utf-8', 'backslashreplace')` will bring any value here.

The line is in a catch clause. What if `data.decode('utf-8'), 'backslashreplace'` still decode error and throws an exception? On the other hand, if the `backslashreplace` decode successfully, the benefit is the log message is more readable, but the whole read content operation already failed, and that readable log message will further obscure real issue.

Reply:
Done